controller/user.controller.js

```diff
@@ -0,0 +1,91 @@
+const { User } = require('../models/User.js');
+
+const getAllUsers = async (req, res) => {
+  try {
+    const users = await User.findAll();
+    res.status(200).json(users);
+  } catch (error) {
+    console.error(error);
+    res.status(500).json({ message: 'Error retrieving users' });
+  }
+};
+
+/*const getUserByID = async (req, res) => {
+  const { user_id } = req.params;
+  try {
+    const user = await User.findByPk(user_id);
```

Comment:
Why is this commented out? Can we verify this is one of the needed rotues in the doc I shared?